unit-tests-java8/src/test/java/org/eclipse/collections/test/map/mutable/MutableMapIterableTestCase.java

```diff
@@ -277,4 +279,18 @@ default void MutableMapIterable_updateValue()
                 Collections.nCopies(1000, 2),
                 map4.values());
     }
+
+    @Test
+    default void Map_forEach()
+    {
+        MutableMapIterable<Object, Integer> map = this.newWith(3, 3, 3, 2, 2, 1);
+        MutableCollection<Integer> forEach = this.newMutableForFilter();
+        map.forEach((key, value) -> forEach.add(value + 10));
+        assertEquals(this.newMutableForFilter(13, 13, 13, 12, 12, 11), forEach);
+
+        MutableMapIterable<Integer, String> map2 = this.newWithKeysValues(3, "Three", 2, "Two", 1, "Three");
+        MutableCollection<String> forEach2 = this.newMutableForFilter();
+        map2.forEach((key, value) -> forEach2.add(key + value));
+        assertEquals(this.newMutableForFilter("3Three", "2Two", "1Three"), forEach2);
+    }
```

Comment:
The prefix matches the class. Putting together the other feedback, this belongs in org.eclipse.collections.test.map.MapIterableTestCase and should be called MapIterable_forEach. We ought to have this in the class MapTestCase, but it looks like we don't have that class yet! Even though we do have org.eclipse.collections.test.list.ListTestCase.

Reply:
For another PR, would it be better to refactor the test cases to have a `MapTestCase` in the style of `ListTestCase`?

Reply:
Yes, it would be great to have MapTestCase.